docker/node/Dockerfile

```diff
@@ -4,7 +4,9 @@ FROM node:8-slim
 
 WORKDIR /app
 
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && apt-get --no-install-recommends install -y \
+    apt-utils \
```

Comment:
`apt-utils` is used for interactive configuration of packages, and is not needed in this environment. Please remove this line.

Reply:
One thing i noticed without "apt-utils" it take longer time to configure the packages and package manager suggest to install "apt-utils"